### DISCUSSION TITLE
Assistance mapping arrow keys?

### DISCUSSION THREAD

**Discussion by toh539py** [nickcoutsos/keymap-editor#239] [Q&A] [answered]
Trying to map the arrow keys on my 60% board. I created a new layer and tried to set my LAlt to layer toggle to the new layer that's mapped with the arrow keys:
![image](https://github.com/nickcoutsos/keymap-editor/assets/70967695/a3cee00c-3073-419b-a584-865ccc02cea6)
![image](https://github.com/nickcoutsos/keymap-editor/assets/70967695/1616747e-c583-47a5-907d-49f5f309789d)

Attempted to create a condition, to no avail:
![image](https://github.com/nickcoutsos/keymap-editor/assets/70967695/0d53a329-b0be-4317-a0f2-3e7ebfb9cdaf)

Basically what I'm hoping to do is create some kind of key combination (LAlt + RCtrl = rt arrow, LAlt + LFn = dn arrow, LAlt + LSuper = lft arrow, and LAlt + LShift = up arrow) where I'm not having to worry about layer switching. Any assistance is very much appreciated!

**Comment by nickcoutsos** [comment, marked as answer]
I think you just want to change that `&to` binding to `&mo` so that holding that key down activates the symbol layer where the other keys are now mapped to arrows.

Conditional layers work a little differently than what I think you're after. With conditional layers you can have a layer activate automatically while a specific set of other layers are active.

**Reply by toh539py** [reply]
Thank you! That was the fix!
